prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/CodeableConcept.yml

```diff
@@ -0,0 +1,181 @@
+coding:
+  generateList: true
+  expressionType: nested
+  vars:
+    code: String, CWE.1
+    display: String, CWE.2
+    system: SYSTEM_URL, CWE.3
+    version: String, CWE.7
+  expressionsMap:
+    code:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CWE.1
+    display:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CWE.2
+    system:
+      type: SYSTEM_URL
+      expressionType: HL7Spec
+      valueOf: CWE.3
+    version:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CWE.7
+    extension_cwe-coding:
+      condition: $code NOT_NULL || $display NOT_NULL || $system NOT_NULL || $version NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
```

Comment:
It makes sense to me here have something separate, but curious if you have thoughts on having a dedicated identifier like this vs. using the `hl7-use` extension?